src/Bluejay.asm

```diff
@@ -1057,33 +1061,48 @@ IF MCU_TYPE == MCU_BB2 or MCU_TYPE == MCU_BB51
     mov  DShot_GCR_Start_Delay, #DSHOT_TLM_START_DELAY
 ENDIF
 
-    setb IE_EA                          ; Enable all interrupts
-
     ; Check if RCP is zero, then it is a normal stop or signal timeout
     jb   Flag_Rcp_Stop, exit_run_mode_no_stall
 
+    ; It is a stall!
     ; Signal stall
     setb Flag_Stall_Notify
 
-    clr  C                              ; Otherwise - it's a stall
+    ; Check max consecutive stalls and exit if stall counter > 3
+    clr  C
     mov  A, Startup_Stall_Cnt
-    subb A, #4                          ; Maximum consecutive stalls
-    jnc  exit_run_mode_stall_done
+    subb A, #3
```

Comment:
Any specific reason you went with 3 here instead of the original 4?

Reply:
I think that 3 tries instead of 4 will heat less the motors and there won't be any difference